src/Storage/ContentRequest/Save.php

```diff
@@ -348,7 +348,9 @@ private function createJsonUpdate(Entity\Content $content, $flush)
 
         $val = $content->toArray();
 
-        if (isset($val['datechanged'])) {
+        if ($val['datechanged'] instanceof Carbon) {
+            $val['datechanged'] = $val['datechanged']->toIso8601String();
+        } elseif (isset($val['datechanged'])) {
```

Comment:
It could… just a very tired team at the end of a year long development cycle :smile: 

You're, or course, most welcome to PR something.